src/LiveComponent/doc/index.rst

```diff
@@ -3672,6 +3672,10 @@ uses Symfony's test client to render and make requests to your components::
                 ->set('count', 99)
             ;
 
+            // Submit form data
```

Comment:
Maybe we should create an entire block "Test LiveComponent with Forms" (or something like that), wdyt ? 

If "no", i think some comment to precise what is "form" and "save" here could avoid some missunderstandings.

Reply:
As Test helper is only one block, I didn't want to change that. But I understand that my array is a bit confusing.

Reply:
Let's keep as it is.. 

Who knows, mabe if you add other nice helpers...  we will be forced to add a dedicated block ? :))